.eslintignore

```diff
@@ -0,0 +1,5 @@
+coverage
+node_modules
+.history
+__fixtures__
+example
```

Comment:
Ideally we wouldn't have this hear, but I was having trouble with wanting to run babel in examples, but not in the root

Reply:
Yeah, its kind of annoying sometimes, I just ignored webpack.config.js in the other project and there was no issues.